src/System.IO.UnmanagedMemoryStream/src/Resources/Strings.resx

```diff
@@ -198,4 +198,7 @@
   <data name="IO_StreamTooLong" xml:space="preserve">
     <value>Stream was too long.</value>
   </data>
+  <data name="Arg_DecBitCtor" xml:space="preserve">
+    <value>Decimal byte array constructor requires an array of length four containing valid decimal bytes.</value>
```

Comment:
Ah, hadn't realized that (was just copying over the old code from mscorlib). Fixed.